tests/test_pyqvm.py

```diff
@@ -380,4 +380,4 @@ def circuit(x, y, z):
             qml.Rot(x, y, z, wires=0)
             return qml.expval(qml.PauliZ(0))
 
-        self.assertAlmostEqual(circuit(a, b, c), np.cos(a) * np.sin(b), delta=3 / np.sqrt(shots))
+        self.assertAlmostEqual(circuit(a, b, c), np.cos(a) * np.sin(b), delta=5 / np.sqrt(shots))
```

Comment:
Rather than increase the std deviation, should we make this test flaky?

Reply:
I have the worry that it would take even more time, currently these tests from `pyqvm.py` take a while to run